packages/google_maps_flutter/google_maps_flutter_ios/example/ios14/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -25,6 +25,6 @@
     <string>arm64</string>
   </array>
   <key>MinimumOSVersion</key>
-  <string>13.0</string>
+  <string>12.0</string>
```

Comment:
Shouldn't this be 14.0?

Reply:
I don't think so; AFAICT this is the plist for the `App.framework` that holds the compiled Dart code, and there's nothing in *that* framework that requires a higher deployment target than Flutter itself. The 13 here was the result of me doing an overzealous find/replace when I make `ios13` from `ios12` (and `ios13` was copied to make the original `ios14`).

Reply:
(The one that matters is the Runner's `Info.plist`, and that is correctly set to 14.0.)

Reply:
sgtm! Thanks!